akka-stream/src/main/scala/akka/stream/impl/Stages.scala

```diff
@@ -126,6 +125,7 @@ private[stream] object Stages {
       Fused(ops, name(ops.iterator.map(x ⇒ Logging.simpleName(x).toLowerCase).mkString("+")))
   }
 
+  //FIMXE this is not used.
```

Comment:
This seems to be a common situation for me, and I fear that it is prohibitive to require searching through the issue database (given its current size) whenever one wants to understand why a piece of code is as it is. Here's my empirical stance on the matter: either the code is there and has a FIXME, or the code is pulled out until the person working on the Issue adds it again. Code that doesn't do anything is a major liability.